README.md

```diff
@@ -12,6 +12,15 @@ Eclipse Instructions
 * In the `Package Explorer`, expand the tree and click on the entry `library->src/test/junit->default package`.
 * Right-click the selected `Package Explorer` entry and click `Run As->JUnit Test`. You should see at least 190 green unit tests, and they should run in a few seconds at most.
 
+Allstate IntelliJ Setup
```

Comment:
this is a public repo, is that ok that this information about your company appears here?